uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -86,6 +86,12 @@ function _hrleaveandabsences_create_administer_menu_tree($leaveAndAbsencesAdminN
           'name'       => 'leave_and_absence_manage_work_patterns',
           'url'        => 'civicrm/admin/leaveandabsences/work_patterns?action=browse&reset=1',
           'permission' => 'administer leave and absences',
+      ),
+      array(
+          'label'      => ts('General Settings'),
```

Comment:
You're using 4 spaces here instead of 2. Please fix this and the other items. Please also use short array syntax for all the items